rxv/rxv.py

```diff
@@ -60,8 +60,8 @@ def __init__(self, play=False, stop=False, pause=False,
 ListGet = '<{src_name}><List_Info>GetParam</List_Info></{src_name}>'
 ListControlJumpLine = '<{src_name}><List_Control><Jump_Line>{lineno}</Jump_Line>' \
                       '</List_Control></{src_name}>'
-ListControlCursor = '<{src_name}><List_Control><Cursor>{action}</Cursor>'\
-                    '</List_Control></{src_name}>'
+ListControlCursor = '<{src_name}><Cursor_Control><Cursor>{action}</Cursor>'\
```

Comment:
this change looks suspicious, it could break things for other people

I need to check this again with real device

Reply:
+1. I got this by watching what the Yamaha Android app transmits. But It could indeed be specific to my receiver (RX-V577). It would be nice if someone could confirm that it works on another model too.

Reply:
I'll check on my RX-V473 :) merging easier PRs first so I can verify everything at once before a new release

Reply:
Ok, so after some years away from rxv I finally started understanding the desc.xml file for real :)

And as such this PR is not ready to be merged.  ListControlCursor and Control_Cursor,Cursor are two completely different features. First one is for browsing trough media, second is, as you mentioned for HDMI/CEC  

see: https://github.com/wuub/rxv/blob/main/tests/samples/rx-v675-desc.xml#L2046 
and for comparison
https://github.com/wuub/rxv/blob/main/tests/samples/rx-v675-desc.xml#L467

I think this is still useful but will need a a small rewrite.
Please let me release what we have now and we'll get back to this

Reply:
Thank you for digging in and finding the difference between these two features! I'll work on fixing this over the next few days. 

> Please let me release what we have now and we'll get back to this

Completely agree. Don't let this PR hold up the release.

Reply:
I added some logic to detect which feature to use to control the cursor based on the input and also added some missing cursor commands for HDMI devices.

I verified that the cursor commands work on the RX-V577 for HDMI inputs and the NET RADIO input.